servicenow/stackstate_checks/servicenow/data/conf.yaml.example

```diff
@@ -22,6 +22,18 @@ instances:
     # Verify https. Default value is true.
     # verify_https: true
 
+    # ServiceNow Custom filter. Check ServiceNow documentation about filtering with sysparm_query parameter.
+    # https://developer.servicenow.com/dev.do#!/learn/learning-plans/orlando/servicenow_application_developer/app_store_learnv2_rest_orlando_more_about_query_parameters
+
+    # ServiceNow CMDB Configuration Items query. Default value is undefined.
+    # cmdb_ci_sysparm_query: company.nameSTARTSWITHaxa
+
+    # ServiceNow CMDB Configuration Items Relations query. Default value is undefined.
+    # cmdb_rel_ci_sysparm_query: parent.company.nameSTARTSWITHaxa^ORchild.company.nameSTARTSWITHaxa
+
+    # ServiceNow Change Request query. Default value is undefined.
```

Comment:
`Default value is undefined` means if `None` no filter will be applied?
can we make the comment a bit more clear?

Also this applies to topology event listing, right? I would also add that in the comment.

Reply:
Added that comment.